wallet-new/src/Cardano/Wallet/Kernel.hs

```diff
@@ -73,12 +104,19 @@ data WalletHandles = Handles {
 
 -- TODO(kde): this will be run with asynchronous exceptions masked.
 -- and we should rethink if migrateMetaDB should happen here.
-handlesOpen :: IO WalletHandles
-handlesOpen = do
-    db <- openMemoryState defDB
-    metadb <- openMetaDB ":memory:" -- TODO: CBR-378
-    migrateMetaDB metadb
-    return $ Handles db metadb
+handlesOpen :: DatabaseMode -> IO WalletHandles
+handlesOpen mode =
+    case mode of
+        UseInMemory -> do
+            db <- openMemoryState defDB
+            metadb <- openMetaDB ":memory:" -- TODO: CBR-378
+            migrateMetaDB metadb
+            return $ Handles db metadb
+        UseFilePath (DatabasePaths macidDb msqliteDb) -> do
+            db <- maybe openLocalState openLocalStateFrom macidDb defDB
+            metadb <- openMetaDB (fromMaybe "./wallet-database/meta.sqlite3" msqliteDb)
```

Comment:
Ah, this is a default that @parsonsmatt put in place in his PR. For the sake of consistency, let me rebrand it to `wallet-db-sqlite` as `wallet-db` is the default if no option is passed.